server/src/test/java/io/crate/integrationtests/MetadataTrackerITest.java

```diff
@@ -372,6 +310,41 @@ public void test_subscribing_to_the_own_tables_on_the_same_cluster() throws Exce
 
     }
 
+    @Test
+    public void test_alter_publication_drop_table_disables_table_replication() throws Exception {
+        executeOnPublisher("CREATE TABLE t1 (id INT) WITH(" + defaultTableSettings() + ")");
+        executeOnPublisher("INSERT INTO t1 (id) VALUES (1), (2)");
```

Comment:
can we add third partitioned table and also exclude it - just to cover all cases?